tests/countries/test_syrian_arab_republic.py

```diff
@@ -0,0 +1,171 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.syrian_arab_republic import SyrianArabRepublic, SY, SYR
+from tests.common import CommonCountryTests
+
+
+class TestSyrianArabRepublic(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1940, 2050)
+        super().setUpClass(SyrianArabRepublic, years=years)
+        cls.no_estimated_holidays = SyrianArabRepublic(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(SyrianArabRepublic, SY, SYR)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(SyrianArabRepublic(years=1939))
+
+    def test_new_year(self):
+        name = "رأس السنة الميلادية"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in self.years))
+
+    def test_mothers_day(self):
+        name = "عيد الأم"
+        self.assertHolidayName(name, (f"{year}-03-21" for year in self.years))
```

Comment:
Same goes for other ones - also make sure that `test_[holiday_name]` test case name matches the `en_US` ones for other entries
```suggestion
    def test_mothers_day(self):
        name = "عيد الأم"
        self.assertHolidayName(name, (f"{year}-03-21" for year in range(1950, 2050)))
        self.assertNoHolidayName(name, range(1947, 1950))
```